modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/views/stories/StoryIntroView.kt

```diff
@@ -13,69 +14,122 @@ import androidx.compose.material.Surface
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawWithContent
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.drawscope.withTransform
 import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
-import androidx.compose.ui.text.intl.Locale
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.tooling.preview.PreviewParameter
 import androidx.compose.ui.unit.dp
 import au.com.shiftyjelly.pocketcasts.compose.AppTheme
+import au.com.shiftyjelly.pocketcasts.compose.parallaxview.ParallaxView
 import au.com.shiftyjelly.pocketcasts.compose.preview.ThemePreviewParameterProvider
 import au.com.shiftyjelly.pocketcasts.endofyear.R
 import au.com.shiftyjelly.pocketcasts.endofyear.components.PodcastLogoWhite
-import au.com.shiftyjelly.pocketcasts.endofyear.components.StoryPrimaryText
-import au.com.shiftyjelly.pocketcasts.repositories.endofyear.stories.StoryIntro
 import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
 import au.com.shiftyjelly.pocketcasts.localization.R as LR
 
 @Composable
 fun StoryIntroView(
-    story: StoryIntro,
     modifier: Modifier = Modifier,
 ) {
-    Column(
-        verticalArrangement = Arrangement.Top,
-        horizontalAlignment = Alignment.CenterHorizontally,
-        modifier = modifier
-            .fillMaxSize()
-            .verticalScroll(rememberScrollState()),
-    ) {
-        Spacer(modifier = modifier.weight(1f))
+    Box {
+        BackgroundImage()
+        Column(
+            verticalArrangement = Arrangement.Top,
+            horizontalAlignment = Alignment.CenterHorizontally,
+            modifier = modifier
+                .fillMaxSize()
+                .verticalScroll(rememberScrollState()),
+        ) {
+            Spacer(modifier = modifier.weight(1f))
 
-        Image(
-            painter = painterResource(R.drawable.img_2022_big),
-            contentDescription = null,
-            modifier = modifier.fillMaxWidth(),
-            contentScale = ContentScale.FillBounds
-        )
+            TitleView()
 
-        Spacer(modifier = modifier.weight(0.4f))
+            Spacer(modifier = modifier.weight(1f))
 
-        PrimaryText(story = story, modifier = modifier)
+            PodcastLogoWhite()
 
-        Spacer(modifier = modifier.weight(1f))
+            Spacer(modifier = modifier.height(30.dp))
+        }
+    }
+}
 
-        PodcastLogoWhite()
+@Composable
+private fun BackgroundImage() {
+    Box(
+        contentAlignment = Alignment.Center,
+    ) {
+        ImageTwenty()
```

Comment:
This seems a bit different from the designs because in the designs the middle numbers numbers (the 0 and the second 2) overlap the text, but we have them here as part of the background, so the text overlaps them.

I think it could be really interesting if we gave each number a slightly different parallax effect so they had different depths (i.e., some were closer and some further away). If you think that sounds interesting @david-gonzalez-a8c  and would be willing to get us separate assets for each number, I'd be interested to see if I could make that look good without having it be too much parallax (or you're more than welcome to Ashita, I just didn't want to add more tasks to your plate 🙂 ).

Here's a quick example of what this could look like with the two groups of numbers we have currently ([code diff](https://gist.github.com/mchowning/c7951f2f00e5b3447b69f5da531b286b)). We'd need to tweak the numbers I'm using for depth if we want some of the numbers to appear in front of the text.


https://github.com/Automattic/pocket-casts-android/assets/4656348/87fa77b1-3edc-4838-8c67-4c89c6e33310

Reply:
> I think it could be really interesting if we gave each number a slightly different parallax effect so they had different depths (i.e., some were closer and some further away).

That'd be awesome! I modified the [specs](https://www.figma.com/file/lH66LwxxgG8btQ8NrM0ldx/End-of-Year?type=design&node-id=1881%3A2951&mode=design&t=DsQRtiw4ou98ocD2-1) so each number can be exported separately, but you can also find them here:

[2023 svg.zip](https://github.com/Automattic/pocket-casts-android/files/12860233/2023.svg.zip)

/cc @leandroalonso